keras_cv/models/object_detection/yolo_v8/yolo_v8_detector.py

```diff
@@ -609,8 +610,13 @@ def decode_predictions(
 
         return self.prediction_decoder(box_preds, scores)
 
-    def make_predict_function(self, force=False):
-        return predict_utils.make_predict_function(self, force=force)
+    def _tf_predict_step(self, data):
```

Comment:
(Same comment re: avoiding multiple definitions of this using the RetinaNet approach)